frontend/public/components/secrets/create-secret.tsx

```diff
@@ -1322,7 +1322,8 @@ export const SecretLoadingWrapper = withTranslation()(
       if (!secretTypeAbstraction) {
         return <LoadingBox />;
       }
-      const fixed = _.reduce(fixedKeys, (acc, k) => ({ ...acc, k: _.get(obj.data, k) }), {});
+      const fixed = fixedKeys.reduce((acc, k) => ({ ...acc, [k]: obj.data?.[k] }), {});
```

Comment:
Is `fixedKeys` always defined?

Reply:
Good catch, @spadgett.  @cyril-ui-developer, might need an update here.  Gonna put a hold on this.

/hold

Reply:
@spadgett @rhamilto Not sure but looking at the [type](https://github.com/openshift/console/blob/7345d545d6060a96b9373df0527e6d074d7dc0b6/frontend/public/components/secrets/create-secret.tsx#L1376) I think so.  But I can add `?` operator it safer.